GameData/RealismOverhaul/RO_SuggestedMods/RealScaleBoosters/RO_RSB_Delta.cfg

```diff
@@ -714,15 +697,45 @@
         }
     }
 
-    !MODULE[ModuleSPU]{}
+    !MODULE[ModuleSPU*],*{}
 
-    !MODULE[ModuleRTAntenna*]{}
+    !MODULE[ModuleRTAntenna*],*{}
 
     !RESOURCE,*{}
+
+    //  Avionics batteries 1.4 kWh.
+    //  Can support the DCSS for flights up to 7 hours in duration.
+    //  Assumes that the electricity consumption of the DCSS avionics is 200 W.
+
+    RESOURCE
```

Comment:
I believe it is preferred that resources like these reside in a module Tank { }, but I'm not sure if that would render this unacceptable. Without being nested in a Tank, the volume couldn't be cleared and reassigned by the user.